packages/apps/crm/src/features/partnerSlice.js

```diff
@@ -69,6 +87,13 @@ const partnerSlice = createSlice({
       state.loadingPartner = false;
       state.partner = action.payload;
     });
+    builder.addCase(searchClientForContact.pending, state => {
+      state.loadingPartner = true;
+    });
+    builder.addCase(searchClientForContact.fulfilled, (state, action) => {
+      state.loadingPartner = false;
+      state.clientForContact = action.payload;
+    });
```

Comment:
Not need to add loading indicator here, you can just fill in clientForContact when request is fullfilled.

I also think it could be better to rename object and functions to linkedPartnersOfContact as it can be propsects too